airflow/dags/unzip_and_validate_gtfs_schedule/convert_to_json/agency.yml

```diff
@@ -0,0 +1,5 @@
+operator: operators.GtfsGcsToJsonlOperator
+
+date: "{{ execution_date }}"
```

Comment:
is this needed? can operators get the execution date in the task context?

Reply:
ope yeah good catch, this was a vestige from a prior attempt. fixed https://github.com/cal-itp/data-infra/pull/1738/commits/7b05d2e3fa8141381c9ee04ea1dd5265d84dcb18